---
title: "Testing pleiotropy vs. separate QTL in multiparental populations"
author: "Frederick Boehm, Brian Yandell, and Karl Broman"
date: "June 20, 2018"
bibliography: research.bib
output:
  xaringan::moon_reader:
    lib_dir: libs
    nature:
      highlightStyle: github
      highlightLines: true
      countIncrementalSlides: false
---

```{r setup, include=FALSE}
options(htmltools.dir.version = FALSE)
library(knitr)
```



---

# Introduction

- Experimentalists can now measure tens of thousands of phenotypes with RNA sequencing and mass spectrometry

- Multiparental populations enable high-resolution QTL mapping

- Together, high-dimensional phenotypes and multiparental populations can inform complex trait genetics

- New analysis tools, such as our test of pleiotropy vs. separate QTL, are needed 


---

# Background

- @jiang1995multiple developed a pleiotropy vs. separate QTL test for F2 mice
    - Applies to two traits that map to a single genomic region 
    - Pleiotropy is the null hypothesis
    - Separate QTL is the alternative hypothesis
    - Perform a two-dimensional QTL scan 
    - Calculate likelihood ratio test statistic


---

# Challenges

- Relatedness    
- Multiple founder lines    
- Test statistic calibration


---

# Solutions to challenges

- Multivariate random effects to account for relatedness    
- Fixed effect for each founder allele    
- Parametric bootstrap for test statistic calibration


---

# Test procedure

- Fit the model: $$Y = XB + G + E$$ for each ordered pair of markers
    - $$G \sim MN(0, K, V_g)$$    
    - $$E \sim MN(0, I, V_e)$$    
    - $X$ contains allele probabilities    
    - $B$ contains allele effects    
- Calculate likelihood for each model fit    
- Test statistic: $$- \log_{10} \frac{\max L_0(B, \Sigma, \omega_1, \omega_2)}{\max L_A(B, \Sigma, \omega_1, \omega_2)}$$    
- Parametric bootstrap to get p-value



---

# Behavioral genetics

- @logan2013high and @recla2014precise genotyped and phenotyped 261 Diversity Outbred mice    
    - Identified *Hydin* as the chromosome 8 gene affecting "hot plate latency" at 57 cM
    - Identified chromosome 8 QTLs for "percent time in light" and "distance traveled in light" at 55 cM    
    - Motivated us to ask if *Hydin* also affects "percent time in light" and "distance traveled in light"    



---

# Allele effects plots

- Suggest that "percent time in light" and "distance traveled in light" share a single QTL
- *Hydin* is distinct from that QTL

```{r, echo = FALSE}
#include_graphics("~/Box Sync/attie/qtl2pleio-manuscript/Rmd/coef_all3.p")
```


---

# Profile LOD plots

- Performed 3 pairwise analyses

```{r, echo = FALSE, out.width="400px"}
include_graphics("figs/all3.png", auto_pdf = FALSE)
```


---

# 3 pairwise tests

- "percent time in light" & "hot plate latency": p = 0.109
- "distance traveled in light" & "hot plate latency": p = 0.108
- "percent time in light" & "distance traveled in light": p = 0.871


---

# Conclusions

- Evidence for two separate QTL affecting the 3 phenotypes
    - 1 QTL affects both "distance traveled in light" and "percent time in light"
    - Second QTL contains *Hydin* and affects "hot plate latency"


---

# Future directions

- Develop methods for analyzing multiple phenotypes in a single test


---

# Contact information

- frederick.boehm@gmail.com  
- https://github.com/fboehm  
- https://github.com/fboehm/qtl2pleio  
- https://fboehm.us/  


---

# References

